pkg/retriever/parallelpeerretriever.go

```diff
@@ -26,12 +26,12 @@ type GetStorageProviderTimeout func(peer peer.ID) time.Duration
 type TransportProtocol interface {
 	Code() multicodec.Code
 	GetMergedMetadata(cid cid.Cid, currentMetadata, newMetadata metadata.Protocol) metadata.Protocol
-	Connect(ctx context.Context, retrieval *retrieval, phaseStartTime time.Time, candidate types.RetrievalCandidate) (time.Duration, error)
+	Connect(ctx context.Context, retrieval *retrieval, startTime time.Time, candidate types.RetrievalCandidate) (time.Duration, error)
 	Retrieve(
 		ctx context.Context,
 		retrieval *retrieval,
 		shared *retrievalShared,
-		phaseStartTime time.Time,
+		startTime time.Time,
```

Comment:
```suggestion
```

try removing this entirely, and from httpretriever.go and graphsyncretriever.go, I don't believe it's used at all in there now

Reply:
This slipped through the cracks. Good catch.